easybuild/easyconfigs/c/cuDNN/cuDNN-7.6.4.38-CUDA-10.0.130.eb

```diff
@@ -0,0 +1,35 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Author:    Stephane Thiell <sthiell@stanford.edu>
+##
+easyblock = 'Tarball'
+
+name = 'cuDNN'
+version = '7.6.4.38'
+local_cuda_version = '10.0.130'
+
+local_cuda_version_major_minor = '.'.join(local_cuda_version.split('.')[:2])
+
+versionsuffix = '-CUDA-%s' % local_cuda_version
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for
+    deep neural networks."""
+
+toolchain = SYSTEM
+
+# Nvidia developer registration required.
+# Download link: https://developer.nvidia.com/rdp/cudnn-download
+# Download as cudnn-10.0-linux-x64-v7.6.5.32.tgz
```

Comment:
this should be cudnn-10.0-linux-x64-v7.6.4.38.tgz

Reply:
Fixed